extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/internal/NtpHandlerFactory.java

```diff
@@ -55,7 +56,7 @@ protected ThingHandler createHandler(Thing thing) {
 
         ThingTypeUID thingTypeUID = thing.getThingTypeUID();
 
-        if (thingTypeUID.equals(THING_TYPE_NTP)) {
+        if (THING_TYPE_NTP.equals(thingTypeUID)) {

```

Comment:
@kaikreuzer How do we like to handle the `@author` section of our files? Should we really add a new author line for such changes that are only "style" and does not bring any new features or real bug fixes?

@AlexanderKostadinov28 Don't get me wrong here, it is not about "adding your name to the author section", but if every line change will be added to that section the section will be bigger then the remaining source code.

Reply:
@maggu2810 I actually wondered and was unsure about the same thing, so I am with you on this as well - I saw contributors being marked for every change and decided to follow this. In my opinion, we should not add contributor lines for every little change in the code. :)